.mailmap

```diff
@@ -1343,6 +1344,7 @@ S. Hanko <suzy.hanko@gmail.com>
 S.Y. Lee <sylee957@gmail.com> (Lazard) S.Y. Lee <sylee957@gmail.com>
 S.Y. Lee <sylee957@gmail.com> Sangyub Lee <sylee957@gmail.com>
 S.Y. Lee <sylee957@gmail.com> sylee957 <sylee957@gmail.com>
+SK <78915702+sskserk@users.noreply.github.com>
```

Comment:
You have added two separate lines in the .mailmap file but it should be one line:
https://github.com/sympy/sympy/blob/d1c962464c97743a3e5ff9a0dc05254cd4434a09/.mailmap#L129-L148